test_data/publications/test_publications.json

```diff
@@ -122,6 +122,17 @@
     "contributor": "test-user-with-author@test.no",
     "title": "Registration with validation error"
   },
+  {
+    "publication_type": "JournalContent",
+    "publication_context_type": "Journal",
+    "publication_instance_type": "JournalArticle",
+    "file_identifier": "lorem_ipsum",
+    "file_name": "lorem_ipsum.txt",
+    "owner": "test-user-view-registration@test.no",
+    "status": "DRAFT",
+    "contributor": "test-user-view-registration@test.no",
+    "title": "View registration"
+  },

```

Comment:
hva mockes egentlig her? ligner mest på registreringssøk, men stemmer ikke helt med typen der heller: https://github.com/BIBSYSDEV/NVA-Frontend/blob/develop/src/types/search.types.ts#L34

Reply:
Det er et sett med verdier jeg legger in i registration for forskjellige testregistreringene. Da slipper jeg å lage en full datastruktur når jeg trenger en ny testregistrering.